lms/djangoapps/learner_dashboard/views.py

```diff
@@ -47,3 +51,53 @@ def program_details(request, program_uuid):
     }
 
     return render_to_response('learner_dashboard/program_details.html', context)
+
+
+@view_auth_classes()
+class ProgramDiscussionIframeView(APIView):
+    """
+    A view for retrieving Program Discussion IFrame .
+
+    Path: ``/dashboard/dashboard/programs/{program_uuid}/iframe/``
+
+    Accepts: [GET]
+
+    ------------------------------------------------------------------------------------
+    GET
+    ------------------------------------------------------------------------------------
+
+    **Returns**
+
+        * 200: OK - Contains a program discussion iframe.
+        * 401: The requesting user is not authenticated.
+        * 403: The requesting user lacks access to the program.
```

Comment:
What access/permissions this view has? can you add explicit permissions to this DRF view?

Reply:
Done